en/config/safety.md

```diff
@@ -227,6 +229,44 @@ Parameter | Description
 [VT_FW_ALT_ERR](../advanced_config/parameter_reference.md#VT_FW_ALT_ERR) | Maximum negative altitude error for fixed wing flight. If the altitude drops more than this value below the altitude setpoint the vehicle will transition back to MC mode and enter failsafe RTL.
 
 
+## Safety Switches {#safety_switch}
+
+A safety switch allows you to immediately crash or return the vehicle from the remote control transmitter
+(if you lose control of the vehicle, this may be better than allowing it to continue flying).
+
+This section lists the available safety switches.
+
+### Kill Switch {#kill_switch}
+
+A kill switch immediately stops all motor outputs, causing the vehicle to crash.
+
+The kill switch is enabled as part of *QGroundControl* [Flight Mode Setup](../config/flight_mode.md#safety_switch).
+
+
+### Arm/Disarm Switch {#arming_switch}
+
+> **Tip** The arming switch is similar to the [kill switch](#kill_switch), but cannot be enabled in all modes during flight (i.e. those modes where it less likely to be needed).
+
+An arm/disarm switch immediately disarms (stop) motors for those [flight modes](../getting_started/flight_modes.md) that *support disarming in flight* (i.e. for these modes it is the same as the [kill switch](#kill_switch)). This includes the modes: 
+- *Manual mode*
```

Comment:
Can you please tell me all the modes in each group? Makes it easier for users.

Reply:
They are: Manual, Acro, Stabilized & Rattitude